prov/gni/src/gnix_dom.c

```diff
@@ -128,8 +128,8 @@ static int gnix_domain_close(fid_t fid)
 	references_held = _gnix_ref_put(domain);
 
 	if (references_held) {
-		GNIX_WARN(FI_LOG_DOMAIN, "failed to fully close domain due to "
-				"lingering references. references=%i dom=%p\n",
+		GNIX_INFO(FI_LOG_DOMAIN, "failed to fully close domain due to "
+			  "lingering references. references=%i dom=%p\n",
 			  references_held, domain);
```

Comment:
should this maybe ```GNIX_WARN```?  Problem with ```GNIX_INFO``` is that for something like OSU benchmarks, bizillions of lines of output can be produced with ```FI_LOG_LEVEL``` set to info.